CITATION.cff

```diff
@@ -37,5 +37,5 @@ keywords:
   - high-resolution
   - iiif
 license: BSD-3-Clause
-version: 5.0.1
+version: 6.0.0
```

Comment:
I usually do this when I actually release, but I suppose it's fine to do it ahead of time.

Reply:
I was just thinking that we talk about osd v6 all the time and you go to the master and see 5.1